Help/Getting_In_Touch.rst

```diff
@@ -24,3 +24,6 @@ Modularity, clean good code as well as a high usability for both users
 and developers of analyse routines (called bears) stand in the
 foreground of the development. We will not speed up our development if
 it needs sacrificing any aspect of quality.
+
+
+Please follow our Code Of Conduct, you can find it `here <http://docs.coala.io/en/latest/Help/FAQ.html#what-is-coala-community-code-of-conduct>`__.
```

Comment:
This page is about contacting people.

This should be after "- Give us feedback", as a new item, 

> - Report any [Code of Conduct](link_to_coc) violations to ....